practices/accessibility-checklist.md

```diff
@@ -83,6 +83,11 @@ Be sure to create an issue or submit a pull request if you have any questions, o
    [WCAG 1, guideline 14](http://www.w3.org/TR/WCAG10/wai-pageauth.html#gl-facilitate-comprehension)  
    (Also see [WCAG 1 core technique 5: "Comprehension"](http://www.w3.org/TR/WCAG10-CORE-TECHS/#comprehension))
 
+1. **Where opening a hyperlink in a new window cannot be avoided, has an advanced warning been provided?**  
+   Unrequested new windows may cause confusion or disorientation for users with cognitive or visual disabilites. Opening links in new windows is discouraged by both WCAG 1.0 and WCAG 2.0. Does the link have a visible marker indicating its behaviour? Is the marker also accessible to screenreaders?  
```

Comment:
Would it make sense to open this up to all users? I feel it's a fair statement, e.g.

> Unrequested new windows can cause confusion or disorientation for many users.